types/three/package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/three",
-    "version": "0.157.9999",
+    "version": "0.158.0",
```

```suggestion
    "version": "0.158.9999",
```

The `.9999` is required; it's to ensure that this package is "newest" in the package graph within the repo for this version (so npm isn't used). This error is unfortunately fatal but we don't error out first thing; will see about fixing that now that loading is lazier.

---

Aha, got it, thanks!

---

Sent https://github.com/microsoft/DefinitelyTyped-tools/pull/789 which would have exposed this sooner.